src/OSPSuite.Infrastructure.Import/Core/DataFormat/MixColumnsDataFormat.cs

```diff
@@ -39,6 +44,7 @@ protected override UnitDescription ExtractUnits(string description, IUnformatted
                .Substring(1, units.Length - 2) //remove the brackets
                .Trim()                         //remove whitespace
                .Split(',')                     //split comma separated list
+               .Where(unitName => _dimensionFactory.DimensionForUnit(unitName) != null) //only accepts valid units
```

Comment:
The same logic should be used for other unit recognition by the way (in column headeR)
Is this the case @abdelr and @georgeDaskalakis